packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -363,6 +365,16 @@ contract Deploy is Deployer {
         initializeOptimismPortal();
     }
 
+    /// @notice Add Plasma setup to the OP chain
```

Comment:
There are a few different ways that this could be done, all with tradeoffs. I don't like having to maintain another config file and having one type of deploy be triggered if a particular value is set in the deploy config

Another way to do it would be a `usePlasma` deploytime config that explicitly says to set up the system in plasma mode

How does the op-node derivation pipeline know its in plasma mode?